src/infrastructure/dataSchemas/plotDataSchema.ts

```diff
@@ -0,0 +1,41 @@
+//NOTE: Includes the actual data collected from a figure: axis values, axis scale, data point values and so
+//NOTE: Do not include any data which depends on StarryDigitizer app(e.g. setting axis by 2 points or 4 points etc...)
+
+import { SemanticVersion } from '@/@types/types'
+//TODO: move definition of type Coord to more global directoly (e.g. @types)
+import { Coord } from '@/domain/models/dataset/datasetInterface'
+
+export interface PlotDataSchema {
```

Comment:
It seems that configDataSchema and PlotDataSchema are not consistently using camelCase and PascalCase.

Reply:
fixed in: https://github.com/t29mato/starry-digitizer/pull/183/commits/9fb1e4b3481d815f49f620015750cea6bf9b9602